jetty-distribution/src/main/resources/modules/fileserver.mod

```diff
@@ -0,0 +1,10 @@
+#
+# File server module
+#
+
+[depend]
+http
```

Comment:
module should not depend directly on http.  It should be able to be used with http2, fcgi, h2 etc.